### PR TITLE
add line to readme

### PR DESCRIPTION
**Pull request by vantage22**

### CODE REVIEW

**Comment by APTy** [comment]
looks good to me

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # senseiLesson2
+new line to readme
```
Comment (APTy) [inline README.md:2]: sup

Comment (vantage22) [reply README.md:2]: i concur

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # senseiLesson2
```
Comment (vantage22) [inline README.md:1]: new comment

**Pull request merged**